astropy/io/fits/connect.py

```diff
@@ -165,6 +165,10 @@ def read_table_fits(input, hdu=None):
     for key, value, comment in table.header.cards:
 
         if key in ['COMMENT', 'HISTORY']:
+            # Convert to astropy's format
```

Comment:
I prefer `io.ascii` which is more specific, otherwise it's difficult to know why exactly "comments".